buckw

```diff
@@ -213,10 +213,15 @@ setupBuckRun ( ) {
 
 # Handle parameters and flags
 handleParams ( ) {
-    # Go directly to kill. Do not run okbuck.
-    if [[ "kill" == $1 ]]; then
-        SKIP_OKBUCK=true
-    fi
+    for arg in "$@"
```

Comment:
I ended up using `printf` so I could split the arrays with new lines. Otherwise `echo` printed them out as one string instead of a list.